bitnami/janusgraph/templates/NOTES.txt

```diff
@@ -67,4 +67,4 @@ To access Janusgraph from outside the cluster follow the steps below:
 
 {{- include "common.warnings.rollingTag" .Values.image }}
 {{- include "janusgraph.validateValues" . }}
-{{- include "common.warnings.modifiedImages" (dict "images" (list .Values.image .Values.metrics.image .Values.volumePermissions.image) "context" $) }}
\ No newline at end of file
+{{- include "common.warnings.modifiedImages" (dict "images" (list .Values.image .Values.metrics.image .Values.volumePermissions.image .Values.cassandra.image) "context" $) }}
```

Comment:
Cassandra is a valid image, isn't it?
https://github.com/bitnami/charts/blob/db33704cd47f8afa9fc0c7822061212668674f42/bitnami/janusgraph/values.yaml#L953